chainerx_cc/chainerx/native/native_device/rounding.cc

```diff
@@ -0,0 +1,16 @@
+#include "chainerx/kernels/rounding.h"
+#include "chainerx/array.h"
+#include "chainerx/native/kernel_regist.h"
+#include "chainerx/numeric.h"
```

Comment:
It looks like these includes aren't sorted. Also, not all of these headers are actually needed here.

Reply:
clang-format formats in this order, even if corrected manually.
Which are the ones not needed?

Reply:
`array.h` is not needed I think.

Reply:
Oops, I thought `out` and `x` are arrays...
Thanks, fixed.